src/main/java/com/joyent/manta/cosbench/config/CosbenchMantaConfigContext.java

```diff
@@ -306,6 +313,15 @@ public Integer getObjectSize() {
         return safeGetInteger("object-size", "Couldn't get object size from COSBench config");
     }
 
+    /**
+     * Reads the configuration and finds the test strategy being benchmarked.
+     *
+     * @return the flag indicating buckets
+     */
+    public String testType() {
+        return safeGetString("test_type", "Couldn't get test_type setting from COSBench config");
```

Comment:
Do we want to default to directories for workload files that don't have a test type specified, so that we support old workload files? I'm not sure what the best approach is.

Reply:
We could do that in order to support backward compatibility, but once *buckets* support, in manta, is completely integrated, we might just use the buckets benchmarking and then it would make more sense to default to buckets. That being said, I feel we should definitely default the ```test_type``` to directories, at least for some near future.